.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -157,4 +167,56 @@ jobs:
         run: |
           kubectl apply -f k8s/website/prod-airqo-website.yaml -n production
           kubectl set image deployment/airqo-website airqo-website=${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-website:${{ needs.image-tag.outputs.build_id }} -n production
-          kubectl annotate deployment/airqo-website kubernetes.io/change-cause="Image updated to ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-website:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n production
\ No newline at end of file
+          kubectl annotate deployment/airqo-website kubernetes.io/change-cause="Image updated to ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-website:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n production
+
+  ### calibrate app ###
+  calibrate-app:
+    name: build-push-deploy-calibrate-app
+    needs: [check, image-tag]
+    if: needs.check.outputs.run_calibrate_app == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Login to GCR
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.REGISTRY_URL }}
+          username: _json_key
+          password: ${{ secrets.GCR_CONFIG }}
+
+      - name: Login to K8S
+        uses: azure/k8s-set-context@v1
+        with:
+          method: kubeconfig
+          kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
+
+      - name: NPM Setup and Build
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: |
+          cd calibrate/
+
+          export REACT_APP_BASE_URL=${{ secrets.REACT_APP_BASE_URL_PROD }}
```

Comment:
@NoahNsimbe, kindly update the repo secrets with `REACT_APP_BASE_URL_PROD` accordingly

Reply:
https://github.com/airqo-platform/AirQo-frontend/pull/574#discussion_r846976467